app/frontend/Calendar/MonthView/MonthView.svelte

```diff
@@ -52,11 +53,14 @@
       }
     }
     let startTime = weekDays[0]; // Always start with Monday
+    let filterStart = startTime.getTime();
```

Comment:
```suggestion
    let filterStart = startTime.clone();
```

Otherwise you compare timestamp with `Date` object below. While that works with coalasing, it's uh.

Reply:
Oh, I see, TypeScript will let you compare `date1 < date2` or `date1.getTime() < date2.getTime()` but not a mixture. I didn't know that.